src/components/Shoutout/Shoutout.jsx

```diff
@@ -30,6 +30,12 @@ const Emoji = ({ id, label, symbol }) => {
     setEmojiClickEnabled(false)
   }
 
+  useEffect(() => {
+    if(emojiCount > 1) { //We will want to change this to be a better number, but this makes it is easy to test
```

Comment:
Ultimately, I think we'll want confetti to fall when emojiCount is equal to some number rather than greater than a number, so we may want to change that part of the condition now. Since we'll want that number to be low for demo purposes, we could just set the condition to be `emojiCount === 2` or something.